glugot/glugot.tce.edu
# GLUGOT site configuration and content

This PR creates a themed GLUGOT site with some content and the relevant Pelican configuration.

**Testing instructions**
* Clone the source branch of this PR locally.
* Create a `virtualenv` environment, activate it, and install the requirements in the `requirements.txt` file.
* Clone the [glugot-pelican-theme repository](https://github.com/glugot/glugot-pelican-theme) under the `themes` directory of this repository with the name `glugot`. Check out the `wip` branch of the theme repository.
* From the root of this repository, run
  ```bash
      $ make devserver
  ```
* The site should be accessible at http://localhost:8000.
* Test and verify that the site looks okay.
* Try adding/modifying content and verify that the site works okay.
